release-notes.md

```diff
@@ -1,7 +1,15 @@
-**Release 1.0.0-ballot de l'Implémentation Guide Service d'Accès aux Soins.**
+**Release 1.1.0-trial-use de l'Implémentation Guide Service d'Accès aux Soins.**
 
-[Modifications apportées dans cette release] (https://github.com/ansforge/IG-fhir-service-acces-aux-soins/milestone/3?closed=1) :
+Modifications apportées dans cette release :
 
-* Mise au format guide d'implémentation FHIR à partir des spécifications PDF existantes (Spécifications agrégation de créneaux et prise de RDV). Première version de ce guide d'implémentation.
-* Ajout des ressources de conformité de la spécification agrégation de créneaux SOS Médecins, sans la partie narrative
-* Ajout de la spécification correspondant au cas d'usage CPTS (ressources de conformité et partie narrative)
+* Prise en compte des retours de la concertation sur la version 1.0.0-ballot
+    - Ajout schéma UML général des ressources en lien avec celles du volet Gestion d'agendas partagés (GAP)
+    - Schéma UML des ressources avec liens vers les profils
+* Amélioration affichage des différents cas d'usage avec gestion des données communes aux cas d'usage
+* Mise au format plant UML de plusieurs schémas
+* Ajout exemples ressource Slot profil CPTS
+* Ajout exemples ressources Appointment
+* Supression des exemples dans le narratif
+* Profilage ressource Practitioner (profilage flux gestion des comptes régulateurs avant ajout du narratif)
+* Changement de version de nos (dépendance) : passage de version 1.2.0 à 1.3.0
+* Passage ven version trial-use suite à fin de concertation CPTS
```

Comment:
Tu peux également rajouter le lien vers l'ensemble des pulls requests qui ont été poussées depuis la dernière release : https://github.com/ansforge/IG-fhir-service-acces-aux-soins/pulls?q=is%3Apr+is%3Aclosed

Je te conseille également, dans chaque PR, de mettre le numéro de version sur laquelle la PR s'applique dans milestone
![Uploading image.png…]()